backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc

```diff
@@ -28,7 +35,8 @@ Restic does not support backing up `hostPath` volumes. For more information, see
 .Prerequisites
 
 * You must install the OpenShift API for Data Protection (OADP) Operator.
-* You must not disable the default Restic installation by setting `spec.configuration.restic.enable` to `false` in the `DataProtectionApplication` CR.
+* You must not disable the default nodeAgent installation by setting `spec.configuration.nodeAgent.enable` to `false` in the `DataProtectionApplication` CR.
+* You must select Kopia or Restic as the uploader by setting `spec.configuration.nodeAgent.enable.uploaderType` to `kopia` or `restic` in the `DataProtectionApplication` CR.
 * The `DataProtectionApplication` CR must be in a `Ready` state.
 
 .Procedure
```

Comment:
Can we have an example DPA CR here, with the nodeAgent configuration highlighted only ?